conf/app.routes

```diff
@@ -15,3 +15,5 @@ GET        /movements/departures/:departureId/messages/:messageId
 POST       /movements/departures/:messageSender/messages/eis                    controllers.NCTSMessageController.post(messageSender: MessageSender)
 
 GET        /movements/departures/:departureId/accompanying-document             controllers.PDFRetrievalController.getAccompanyingDocument(departureId: DepartureId)
+
+GET        /test                                                                controllers.DeparturesController.test
```

Comment:
Kind of goes without saying but...don't forget to remove this endpoint 😛

Reply:
Well this is just to demonstrate the code, not to be seriously merged - but yes. Agreed